CMakeLists.txt

```diff
@@ -113,7 +116,7 @@ elseif (${UNIX})
 
     set(PY_SHORT_VERSION "${PY_VERSION_MAJOR}${PY_VERSION_MINOR}")
 
-    find_package(Boost 1.71 COMPONENTS filesystem python${PY_SHORT_VERSION})
+    find_package(Boost 1.71 COMPONENTS system filesystem python${PY_SHORT_VERSION})
```

Comment:
Is this necessary for Unix systems?

Reply:
Yes. Because it links for linux also. But current version of boost find it anyway. This is a fix to support any version of boost

Reply:
Then it should probably also be added in the next line: `find_package(Boost 1.65 REQUIRED COMPONENTS filesystem python-py${PY_SHORT_VERSION})`